File: backend/src/main/java/com/bookworm/service/AuthService.java
package main.java.com.bookworm.service;

import com.bookworm.model.User;
import com.bookworm.repository.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class AuthService {

    @Autowired
    private UserRepository userRepository;

    public String login(String username, String password) {
        User user = userRepository.findByUsername(username);
        if (user != null && user.getPassword().equals(password)) {
            return "Login Successful";
        }
        return "Invalid Credentials";
    }
}

File: frontend/src/services/AuthService.js
import axios from 'axios';

const API_URL = '/api/auth';

const login = (username, password) => axios.post(`${API_URL}/login`, { username, password });
const register = (username, password) => axios.post(`${API_URL}/register`, { username, password });

const AuthService = { login, register };
export default AuthService;
